worker/worker.go

```diff
@@ -29,6 +29,24 @@ type Worker struct {
 	QueueId        string
 }
 
+type TaskState uint8
+
+const (
+	TaskPending TaskState = iota
+	TaskRunning
+	TaskFinished
+	TaskFailed
+)
+
+type Task struct {
```

Comment:
this is to be merged with a previous PR, right?

Reply:
yep